### PR TITLE
Fixes napalm grenades failing to detonate if still held after their timer expires, fixes description of Napalm in uplink, and reduces cost by three

### LINKED ISSUE #18046: [Bug]: Napalm grenades never go off.
### Checks

- [X] I have searched the bug with a few keywords, and I confirm this bug was not yet reported.

### Location

Horizon

### Describe the issue

When primed, the napalm grenades never go off, they just remain in the primed but not exploded state (blinking light, lever released)

### How to reproduce

1. Get Napalm grenade
2. prime napalm grenade
3. nothing happens

### Round ID

_No response_

### Staff ckey

_No response_

### PR DESCRIPTION
**Pull request by AlaunusLux**
Also changes the description in the uplink, changes the cost to five (from eight).

Fixes #18046

### CODE REVIEW

**Comment by AlaunusLux** [comment]
!bugfix

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 1]
```diff
@@ -491,11 +491,11 @@
 
 /obj/item/storage/box/grenades/napalm
 	name = "box of napalm grenades"
-	desc = "A box containing 3 napalm grenades."
+	desc = "A box containing 5 napalm grenades."
 	icon_state = "secbox"
 	item_state = "secbox"
 	illustration = "grenade"
-	starts_with = list(/obj/item/grenade/napalm = 3)
+	starts_with = list(/obj/item/grenade/napalm = 5)
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/storage/boxes.dm:498]: The uplink description is wrong, they should be 3 grenades, not 5, as napalm is quite long-lasting

Comment (AlaunusLux) [reply code/game/objects/items/weapons/storage/boxes.dm:498]: Is 8 TC the intended cost for three grenades?

Comment (FluffyGhoster) [reply code/game/objects/items/weapons/storage/boxes.dm:498]: It was, but it can be tuned too

**Comment by AlaunusLux** [comment]
!review

**Comment by NonQueueingMatt** [comment]
Not a bugfix if you're changing balance values. Also, change the title to reflect your entire PR.

**Review by FluffyGhoster — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**